overlord/snapstate/snapstate.go

```diff
@@ -47,9 +47,13 @@ const (
 	// always be enforcing, even if the snap requests otherwise.
 	JailMode
 
-	// if we need flags for just SnapSetup it may be easier
-	// to start a new sequence from the other end with:
-	// 0x40000000 >> iota
+	// flags just for SnapSetup follow
```

Comment:
you actually need a new const ( ) block and possibly to  use the specific type for SnapSetup flags,

if you keep going in this block you don't get what you want I think (iota will not start from 0 again)